packages/next/server/node-web-streams-helper.ts

```diff
@@ -7,6 +7,8 @@ export type ReactReadableStream = ReadableStream<Uint8Array> & {
 export function readableStreamTee<T = any>(
   readable: ReadableStream<T>
 ): [ReadableStream<T>, ReadableStream<T>] {
+  if (readable.tee) return readable.tee()
```

Comment:
I'm not sure if this works on CF TBH...

Reply:
I'll remove it for now and to see if we can fully replace this helper later